### PR TITLE
Update/v8 auth browser

### PR DESCRIPTION
**Pull request by claireriley**
Removed images + text for browser authentication flow based on update

`<!-- -->` is for spacing between two block quotes

### CODE REVIEW

File: src/pages/docs/getting-started/postman-account.md [hunk 1]
```diff
@@ -110,12 +110,9 @@ When you opt to sign into the desktop version of Postman, you will see the follo
...
-Click **Continue** to move forward with logging into your account.
-
+<!-- -->
 > You must complete the process of signing in within five minutes once you initiate login from Postman. If you go beyond this time, you must return to Postman and restart the sign in process.
 
-<img src="https://assets.postman.com/postman-docs/sign-in-to-postman-continue.jpg" width="350px" alt="Sign in to Postman"/>
-
 Log into Postman in your browser by entering your account credentials or signing in with Google. If you are on a Postman Business or Enterprise plan, opt to **Sign in with Single Sign-On (SSO)**.
 
 > Check **Keep me signed in** if you would like to remain signed in after your current session for 30 days before re-authenticating. If you do not want to remain signed in on the computer you are working on, uncheck this option. Note that you will be prompted to sign in again after 30 minutes.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/getting-started/postman-account.md:118]: The screenshot here should be the one generated from the app onto the browser. It should have a message specifying "You're redirected from the Postman desktop app on OS X 18.0.7 to sign in"

image

image

File: src/pages/docs/administration/sso/user-sso.md [hunk 2]
```diff
@@ -27,12 +27,8 @@ When you opt to sign into the desktop version of Postman, you will see the follo
...
 
-Click **Continue** to move forward with logging into your account.
-
 > You must complete the process of signing in within five minutes once you initiate login from Postman. If you go beyond this time, you must return to Postman and restart the sign in process.
 
-<img src="https://assets.postman.com/postman-docs/sign-in-to-postman-continue.jpg" width="350px" alt="Sign in to Postman"/>
-
 Log into Postman by selecting **Sign in with Single Sign-On (SSO)** in the upper-right.
 
 <img src="https://assets.postman.com/postman-docs/sign-in-full-page.jpg" alt="Sign in"/>
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/sso/user-sso.md:34]: The screenshot here should be the ones generated from the app. It should have a message specifying
"You're redirected from the Postman desktop app on OS X 18.0.7 to sign in"

File: src/pages/docs/getting-started/postman-account.md (hunk 1 above)
Comment (ArjunSingh-PM) [reply src/pages/docs/getting-started/postman-account.md:118]: Also, check if you see the 'account chooser' screen before you see the sign-in screen. 

Add account (app) -> account chooser (browser) -> add account (browser) -> sign in screen (browser)

**Review by ArjunSingh-PM — APPROVED** [review]
Need to replace the screenshots generated from the app. Looks good otherwise!

**Review by sowmyaraj92 — APPROVED** [review]
@claireriley We are good to merge! 🎉

**Pull request merged**